components/form/ErrorLabel.js

```diff
@@ -0,0 +1,16 @@
+import { useFormContext } from 'react-hook-form';
+export function ErrorLabel({ className, name }) {
+  const {
+    formState: { errors }
+  } = useFormContext();
+
+  return (
+    <>
+      {errors?.[name] && (
+        <div className={`text-amcovad-danger py-1 ${className} `}>
+          <p>{errors?.[name]?.message}</p>
```

Comment:
The `p` tag is not needed here.

```suggestion
          {errors?.[name]?.message}
```